code/game/objects/structures/headpike.dm

```diff
@@ -0,0 +1,42 @@
+/obj/structure/headpike
+	name = "spooky head on a spear"
+	desc = "When you really want to send a message."
+	icon = 'icons/obj/structures.dmi'
+	icon_state = "headpike"
+	density = FALSE
+	anchored = TRUE
+
+/obj/structure/headpike/bone //for bone spears
+	icon_state = "headpike-bone"
+
+
+/obj/structure/headpike/CheckParts(list/parts_list)
+	..()
+	var/obj/item/bodypart/head/H = locate() in contents
+	if(H)
+		copy_overlays(H, TRUE) //copy the head's visuals to put on the spear
```

Comment:
```
var/mutable_appearance/MA = new()
MA.overlays = H.overlays.Copy()
MA.pixel_y = 12 //Or some value that lines it up on the spike bit better
add_overlay(MA)
```

This will fix the head being in the middle, and put it on the top.